scripts/index.js

```diff
@@ -0,0 +1,92 @@
+// BUTTON EVENTS
+// The object state is accessible in all our functions
+
+const state = {
+  currentTemp: 71
+};
+
+const updateTemperature = (currentTemp) => {
+    const tempValue = document.querySelector("#temp_value");
+    tempValue.textContent = currentTemp + ` °F`;
+    const weatherLandscape= document.querySelector("#landscape_picture");
+    if (currentTemp >= 80) {
+        weatherLandscape.src = "https://freepngimg.com/thumb/lizard/27295-6-lizard-transparent-background-thumb.png";
+        tempValue.className = "red";
+    } else if (currentTemp >= 70) {
+        weatherLandscape.src = "https://freepngimg.com/thumb/chicken/17-chicken-png-image-thumb.png";
+        tempValue.className = "orange";
+    } else if (currentTemp >= 60) {
+        weatherLandscape.src = "https://freepngimg.com/thumb/fox/10-2-fox-png-6-thumb.png";
+        tempValue.className = "yellow";
+    } else if (currentTemp >= 50) {
+        weatherLandscape.src = "https://freepngimg.com/thumb/deer/93967-springbok-antelope-africa-vecteur-free-download-png-hq-thumb.png";
+        tempValue.className = "green";
+    } else if (currentTemp >= 40) {
+        weatherLandscape.src = "https://freepngimg.com/thumb/polar_bear/29346-2-polar-bear-image-thumb.png";
+        tempValue.className = "teal";
+    }
+}
+
+const increaseTemp = () => {
+    state.currentTemp += 1;
+    updateTemperature(state.currentTemp);
+};
+
+const decreaseTemp = () => {
+    state.currentTemp -= 1;
+    updateTemperature(state.currentTemp);
+};
+
+const changeWeather = () => {
+    const weatherMenu = document.querySelector("#weather_menu").value;
+    const weatherIcon = document.querySelector("#weather_picture");
+    if (weatherMenu === "Sunny") {
+        weatherIcon.src = "https://cdn4.iconfinder.com/data/icons/the-weather-is-nice-today/64/weather_3-256.png";
+    } else if (weatherMenu === "Cloudy") {
+        weatherIcon.src = "https://cdn4.iconfinder.com/data/icons/the-weather-is-nice-today/64/weather_1-256.png";
+    } else if (weatherMenu === "Rainy") {
+        weatherIcon.src = "https://cdn4.iconfinder.com/data/icons/the-weather-is-nice-today/64/weather_16-256.png";
+    } else if (weatherMenu == "Snowy") {
+        weatherIcon.src = "https://cdn4.iconfinder.com/data/icons/the-weather-is-nice-today/64/weather_35-256.png";
+    } else if (weatherMenu === "Empty") {
+        weatherIcon.src = "";
```

Comment:
This is a smart way of populating the `image src`. My one critique is that empty image icons infer that there's a broken image source which can confuse users into thinking it was their browser that caused it before even interacting with your page. Having some type of placeholder instead of an empty image source will improve this user experience.

Reply:
Thank you, Audrey, I will keep user experience in mind!